lib/news_scraper/configuration.rb

```diff
@@ -0,0 +1,16 @@
+module NewsScraper
+  class Configuration
+    DEFAULT_SCRAPE_PATTERNS_FILEPATH = File.expand_path('../../../config/article_scrape_patterns.yml', __FILE__)
+    attr_accessor :scrape_patterns, :scrape_patterns_filepath
```

Comment:
Well, in a future iteration - it might be one or the other is settable since scraping doesnt need a filepath